src/test/py/bazel/bzlmod/mod_command_test.py

```diff
@@ -454,6 +455,64 @@ def testShowRepoThrowsUnusedModule(self):
         stderr,
     )
 
+  def testDumpRepoMapping(self):
+    _, stdout, _ = self.RunBazel(
+      [
+        'mod',
+        'dump_repo_mapping',
+        '',
```

Comment:
oooh, clever :)

Reply:
I'm not scared of empty strings (much less so than `_main`) :-)